### PR TITLE
Update j from 806 to 807

### PR DESCRIPTION
**Pull request by 00dani**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by 00dani** [comment]
`cask-repair` *seemed* to work entirely automatically, but the `updatejqt.sh` install script from the previous version was removed in this version for some reason. So I've patched that issue manually. It installs fine now. 🐱

**Comment by vitorgalvao** [comment]
> `cask-repair` _seemed_ to work entirely automatically, but the `updatejqt.sh` install script from the previous version was removed in this version for some reason.

That could only be detected by performing an installation (and even then, there’s only so much a bash script can do). If I made `cask-repair` install the cask every time it updates it, I’d get a lot of confused and angry people. That would definitely not be worth it for the minority of cases a tweak is needed (which I’d bet is close to or even below 1%).

File: Casks/j.rb [hunk 1]
```diff
@@ -11,7 +11,10 @@
     app "j64-#{version}/#{a}.app"
   end
 
-  installer script: "j64-#{version}/updatejqt.sh"
+  installer script: {
+                      executable: "j64-#{version}/bin/jconsole",
+                      args: ['-js', "exit install'qtide'"]
```
Comment (vitorgalvao) [inline Casks/j.rb]: Does it work instead as:

```suggestion
                      args: ['-js', 'exit install"qtide"']
```

Comment (00dani) [reply Casks/j.rb]: Unfortuntely no. Strings in J can only use single quotes, since `"` is instead an operator.

**Comment by 00dani** [comment]
> That could only be detected by performing an installation (and even then, there’s only so much a bash script can do). 

Oh, sure, absolutely! It's quite reasonable for `cask-repair` to work the way it does. I'm mostly just confused that they removed that script from the upstream package, even though you still need to do the things it does to install Jqt properly.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**